notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>Noteworthy that sy.networks[0] logs in as a guest user to the network. Should we introduce this concept yet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
To clarify and @shubham3121 correct me if I'm misreading this but it's logging them in as a guest user to the domain not the network. So this is the same scenario as above just using the network as a proxy to find the domain.

Reply:
<p>Yes, the statement <code>network_client = sy.networks[0]</code>  (this is already implemented) logs a user as  a guest a user and is used as a proxy to find the domain. Otherwise, no other operation can done on the network node.</p>

Reply:
<p>The <code># Call the .register method</code> comment should probably include information about what the <code>.register</code> method is going to do (register with the website?). Love the inclusion of potential calls they might want to make as a guest user.  </p>